include/zephyr/bluetooth/uuid.h

```diff
@@ -1371,203 +2658,1362 @@ struct bt_uuid_128 {
+	BT_UUID_DECLARE_16(BT_UUID_GATT_ACS_VAL)
 /**
- *  @brief OTS Object Action Control Point Characteristic UUID value
+ *  @brief GATT Characteristic Physical Activity Session Descriptor UUID Value
  */
-#define BT_UUID_OTS_ACTION_CP_VAL 0x2ac5
+#define BT_UUID_GATT_PHY_ASDESC_VAL 0x2b45
 /**
- *  @brief OTS Object Action Control Point Characteristic
+ *  @brief GATT Characteristic Physical Activity Session Descriptor
  */
-#define BT_UUID_OTS_ACTION_CP \
-	BT_UUID_DECLARE_16(BT_UUID_OTS_ACTION_CP_VAL)
+#define BT_UUID_GATT_PHY_ASDESC \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_PHY_ASDESC_VAL)
 /**
- *  @brief OTS Object List Control Point Characteristic UUID value
+ *  @brief GATT Characteristic Preffered Units UUID Value
  */
-#define BT_UUID_OTS_LIST_CP_VAL 0x2ac6
+#define BT_UUID_GATT_PREF_U_VAL 0x2b46
 /**
- *  @brief OTS Object List Control Point Characteristic
+ *  @brief GATT Characteristic Preffered Units
  */
-#define BT_UUID_OTS_LIST_CP \
-	BT_UUID_DECLARE_16(BT_UUID_OTS_LIST_CP_VAL)
+#define BT_UUID_GATT_PREF_U \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_PREF_U_VAL)
 /**
- *  @brief OTS Object List Filter Characteristic UUID value
+ *  @brief GATT Characteristic High Resolution Height UUID Value
  */
-#define BT_UUID_OTS_LIST_FILTER_VAL 0x2ac7
+#define BT_UUID_GATT_HRES_H_VAL 0x2b47
 /**
- *  @brief OTS Object List Filter Characteristic
+ *  @brief GATT Characteristic High Resolution Height
  */
-#define BT_UUID_OTS_LIST_FILTER \
-	BT_UUID_DECLARE_16(BT_UUID_OTS_LIST_FILTER_VAL)
+#define BT_UUID_GATT_HRES_H \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_HRES_H_VAL)
 /**
- *  @brief OTS Object Changed Characteristic UUID value
+ *  @brief GATT Characteristic Middle Name UUID Value
  */
-#define BT_UUID_OTS_CHANGED_VAL 0x2ac8
+#define BT_UUID_GATT_MID_NAME_VAL 0x2b48
 /**
- *  @brief OTS Object Changed Characteristic
+ *  @brief GATT Characteristic Middle Name
  */
-#define BT_UUID_OTS_CHANGED \
-	BT_UUID_DECLARE_16(BT_UUID_OTS_CHANGED_VAL)
+#define BT_UUID_GATT_MID_NAME \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_MID_NAME_VAL)
 /**
- *  @brief OTS Unspecified Object Type UUID value
+ *  @brief GATT Characteristic Stride Length UUID Value
  */
-#define BT_UUID_OTS_TYPE_UNSPECIFIED_VAL 0x2aca
+#define BT_UUID_GATT_STRDLEN_VAL 0x2b49
 /**
- *  @brief OTS Unspecified Object Type
+ *  @brief GATT Characteristic Stride Length
  */
-#define BT_UUID_OTS_TYPE_UNSPECIFIED \
-	BT_UUID_DECLARE_16(BT_UUID_OTS_TYPE_UNSPECIFIED_VAL)
+#define BT_UUID_GATT_STRDLEN \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_STRDLEN_VAL)
 /**
- *  @brief OTS Directory Listing UUID value
+ *  @brief GATT Characteristic Handedness UUID Value
  */
-#define BT_UUID_OTS_DIRECTORY_LISTING_VAL 0x2acb
+#define BT_UUID_GATT_HNDDNSS_VAL 0x2b4a
 /**
- *  @brief OTS Directory Listing
+ *  @brief GATT Characteristic Handedness
  */
-#define BT_UUID_OTS_DIRECTORY_LISTING \
-	BT_UUID_DECLARE_16(BT_UUID_OTS_DIRECTORY_LISTING_VAL)
+#define BT_UUID_GATT_HNDDNSS \
+	BT_UUID_DECLARE_16(BT_UUID_GATT_HNDDNSS_VAL)
```

Comment:
Is `HNDDNSS` an abbreviation from the SIG, or one that you came up with? 

IMO this should just be BT_UUID_GATT_HANDEDNESS

Reply:
Fixed